src/style/document-header.less

```diff
@@ -59,13 +59,54 @@
 .document-header {
     height: @toolbar-button-size;
     line-height: @toolbar-button-size;
-    max-width: 100%;
-    width: 100%;
+    flex-grow: 10;
     position: relative;
     overflow: hidden;
     display: flex;
 }
 
+.generator-header {
```

Comment:
this is a nit that I should have caught before, but I don't really like the use of `generator`.  I'd prefer `export-`.   I'm on the fence as to whether or not this is a must-fix.